src/components/KPop/KPop.vue

```diff
@@ -470,7 +475,7 @@ $kPopCaretOffset: 16px;
       // fixing mixed-decls deprecation: https://sass-lang.com/d/mixed-decls
       // stylelint-disable-next-line no-duplicate-selectors
       & {
-        margin-bottom: var(--kui-space-60, $kui-space-60);
+        margin-bottom: v-bind('popoverOffset');
```

Comment:
I think there's a chance using v-bind here may cause hydration errors in portal?